src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -585,14 +585,14 @@ public Customer(string value)
         // {ServicesVSResources.Prefer_colon}
         int fibonacci(int n)
         {{
-            return n <= 1 ? 1 : fibonacci(n - 1) + fibonacci(n - 2);
+            return n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
```

Comment:
💭 As a best guess, this was mixed up with the base cases for a recursive definition of x! (factorial) in the original implementation

Reply:
It wasn't wrong before. See http://whatis.techtarget.com/definition/Fibonacci-sequence

Reply:
Also https://en.wikipedia.org/wiki/Fibonacci_number

Reply:
but no matter whether you start from 1 or 0, fibonacci(2) is 1, not 2

Reply:
it would have returned 2

Reply:
Also http://mathworld.wolfram.com/FibonacciNumber.html and https://www.mathsisfun.com/definitions/fibonacci-sequence.html

Reply:
but in all those links it literally says f(1) is 1 and f(2) is 1, so what do you mean? i don't get it

Reply:
it would have returned f(2)=2 which is wrong

Reply:
f(0) is supposed to be 0. f(1) is supposed to be 1. f(2) will be 1 by addition of 0 and 1.

the PR's change makes f(0) 1 instead of 0, and makes f(2) 2 instead of 1, which disagrees with modern convention for the sequence.

Reply:
it also says f(0) may optionally be defined as 0, but f(2)=1 regardless

Reply:
no it doesn't? f(0) will be n<=1 therefore n therefore 0, previously it would have returned f(0)=1

Reply:
i think you are confusing the diff

Reply:
Oh, yes, I'm reading the diff backwards!

Reply:
no before it was `n <= 1 ? 1` , not it is `n <= 1 ? n` so before it would have returned f(0) = 1, now it will return 0